sdk/keyvault/Azure.Security.KeyVault.Secrets/api/Azure.Security.KeyVault.Secrets.net8.0.cs

```diff
@@ -144,7 +144,6 @@ public static partial class SecretClientBuilderExtensions
     {
         public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Security.KeyVault.Secrets.SecretClient, Azure.Security.KeyVault.Secrets.SecretClientOptions> AddSecretClient<TBuilder>(this TBuilder builder, System.Uri vaultUri) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilderWithCredential { throw null; }
         [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("Binding strongly typed objects to configuration values requires generating dynamic code at runtime, for example instantiating generic types. Use the Configuration Binder Source Generator (EnableConfigurationBindingGenerator=true) instead.")]
-        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Binding strongly typed objects to configuration values is not supported with trimming. Use the Configuration Binder Source Generator (EnableConfigurationBindingGenerator=true) instead.")]
```

Comment:
Is this intentional?

Reply:
Yes, this is a side effect of System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute being added to the ApiListing.exclude-attributes.txt in https://github.com/Azure/azure-sdk-for-net/pull/48102/files#diff-4c3e40a00dbe39daab6f947a3910821268e302fc338aa68762630745f23a868eR1
keyvault already had it generated into the api listing before it was added.

Reply:
I don't think it was caught in that PR since the keyvault ci pipeline didn't run, but I just fixed it here when I saw the failure

Reply:
Thanks